app/javascript/show_search_fields.js

```diff
@@ -0,0 +1,39 @@
+document.addEventListener('turbo:load', () => {
+  const searchField = document.getElementById('search-field')
+  setupContainer(searchField)
+})
+
+const setupContainer = (searchField) => {
+  searchField.addEventListener('keydown', () => {
+    const searchText = document.getElementById(searchField.id).value
+    const searchContainer = document.getElementById('search-container')
+    searchContainer.classList.remove('hidden')
+    if (searchText.length > 2) {
+      const ul = document.getElementById('patients-list');
+      ul.innerHTML = ""
+      const patients = httpGet(searchText)
+      setupPatientsList(patients)
+    } else {
+      searchContainer.classList.add('hidden')
+    }
+  })
+}
+
+const httpGet = (searchText) => {
+  var xmlHttp = new XMLHttpRequest()
```

Comment:
Le's use [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) instead
Also why does `xmlHttp` has to be `var`?